source/RialtoGStreamerMSEAudioSink.cpp

```diff
@@ -166,6 +166,31 @@ rialto_mse_audio_sink_create_media_source(RialtoMSEBaseSink *sink, GstCaps *caps
                 return nullptr;
             }
         }
+        else if (g_str_has_prefix(strct_name, "audio/b-wav"))
+        {
+            gint sample_rate = 0;
+            gint number_of_channels = 0;
+            std::optional<uint64_t> channelMask;
+            gst_structure_get_int(structure, "rate", &sample_rate);
```

Comment:
Should we print a warning if gst_structure_get_int returns false? (and similar for the channels beneath?) These fields seem important/ mandatory (maybe they aren't?)